script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -284,16 +284,16 @@ I was glad. For the first time since I woke up from the accident, I felt human a
 <Page78>
 It became my routine to go to the field in the afternoon.
 The woman would get upset if I called her Aoko. She said she didn't like her name.
-After much pondering, I decided to call her "Sensei" because she seemed to be an important person.
+After much pondering, I decided to call her 'Sensei' because she seemed to be an important person.
 <Page79>
 Sensei listened to me seriously no matter what I said and solved my problems with just a few words.
 <Page80>
 After the accident, I had become uncomfortable talking to people, but thanks to her, I was able to gradually return to my normal self.
 <Page81>
-Only when I was in this field did my anxiety disappear because I knew that I could be my normal self here.
+Only after I was in this field did my anxiety disappear because I realized I could be myself.//Shinyyyjoke suggestion
```

Comment:
Is the jp text "_while_ I was in this field I did not feel anxiety", as in he is non anxious when not in the field, or is it that he is _never_ anxious after being there once?

Reply:
> この野原にいる時だけは

only while I was in this field. 

I think original is better